engine/chain/interface.go

```diff
@@ -15,6 +17,7 @@ type ARCService interface {
 
 // BHSService for querying BHS server.
 type BHSService interface {
+	GetMerkleRootsFromBHS(ctx context.Context, query url.Values) (*models.MerkleRootsBHSResponse, error)
```

Comment:
I would not add the `FromBHS` suffix as other methods (within BHSSErvice or ARCService) don't have such suffix. 
So I vote for `GetMerkleRoots`.